liubai-docs/docs/guide/intelligent-pocket.md

```
@@ -0,0 +1,35 @@
+# 智能口袋
+
+微信提供 `置顶服务号` 和 `浮窗` 两个功能。搭配留白记事使用，会变得非常好用！
+
+<img src="./assets-community/weixin.png" alt="留白记事微信公众号" width="330">
+
+## 置顶服务号
+
+扫描上方二维码关注服务号，点击右上角的 `...` → `置顶服务号`：
+
+<img src="./assets-intelligent-pocket/01.png" width="400" />
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)